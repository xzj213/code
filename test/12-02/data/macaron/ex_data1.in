3
5 9 15
19 7 2
1 0 0
