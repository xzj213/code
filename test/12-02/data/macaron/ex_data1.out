2 7 9 10 26 28
